.babelrc

```diff
@@ -1,9 +1,15 @@
 {
-  "presets": ["env", "react", "flow"],
+  "presets":    [
+    [ "env", {
+      "modules": "commonjs",
+      "useBuiltIns": true
+    }],
+    "react",
+    "flow"
+  ],
   "plugins": [
   	"transform-class-properties",
-  	"transform-object-rest-spread",
-  	"lodash"
```

Comment:
Not needed anymore as lodash is an external dependency in webpack now.

Reply:
But that's the babel-lodash-plugin, it doesn't have anything to do with dependencies.

Reply:
It does. it converts all of the `lodash` imports to `lodash/<file>` which then screws with the externals in the webpack config and then builds locash into each entry (which we do not want... :-( )